xla/service/gpu/gpu_compiler.cc

```diff
@@ -1216,7 +1217,11 @@ absl::Status RunPostFusionSimplificationPasses(
         gpu_target_config.device_description);
     pipeline.AddPass<StreamAttributeAsyncWrapper>();
   }
-
+  if (hlo_module->config()
```

Comment:
Can you add an execution test for this? Will the runtime be able to deal with the async ops you create here?

Reply:
I was planning to add the execution test to https://github.com/openxla/xla/pull/20155 once this PR is merged.